Zachary Williams: first post and _config.yml update

zbay opened the pull request:
Hello, @eah13. Here is my pull request for the edit to my config.yml and my first post.
Here's the link to my homepage: http://zbay.github.io/spring2014/

Fixes #2 

Fixes #3

zbay committed:
Update _config.yml

Forked with my author info.

zbay committed:
Update _config.yml

Forked with my author info.

zbay committed:
My first Jekyll post

Check it out, y'all.

zbay committed:
Rename 2014-01-08-firstpostzbay to 2014-01-08-firstpostzbay.md

Changed extension to .md

zbay committed:
Update 2014-01-08-firstpostzbay.md

Added spaces between header information.

eah13 commented:
Zachary,

Make sure to include a link to your page with your pull requests.  If you visit it, you'll see the stie is broken:
http://zbay.github.io/spring2014/

I'll leave some comments in the diff to discuss why. Please get this resolved quickly so you don't fall behind.

eah13 commented on _config.yml:
```diff
@@ -15,19 +15,11 @@ course:
-      website: http://grantmclendon.com
-      github: gerbal
-      twitter: gerbal100
-      about: Grant is a first year MSIS student. He's from Chapel Hil and like playing with new technologies and getting way in over his head.
```
@zbay you can see here you've deleted Grant and I from the config.  This prevents the site from knowing who we are.  Please add us back.  To do this you'll want to copy and paste our information (and ours only, not that of other students) from here:
https://raw2.github.com/silshack/spring2014/gh-pages/_config.yml

eah13 commented on _posts/2014-01-08-firstpostzbay.md:
```diff
@@ -0,0 +1,22 @@
+- - -
```
This line and the one below have spaces in between the hyphens.  The three hyphens shouldn't have any spaces.  Little differences like this will confuse Jekyll, so make sure to follow directions closely.

zbay committed:
Re-added config's original authors

I misunderstood _config.yml's role and removed its original authors. I re-added them in this edit and added a little bit of detail to my bio.

zbay committed:
Removed spaces between header hyphens

I mistakenly put spaces between the six hyphens that make up a post's header. It won't happen again.

zbay commented:
I fixed the problems you noted, @eah13, and verified my Github email address. Hopefully I'm on track now.

eah13 commented:
@zbay did you check your version of the site?: http://zbay.github.io/spring2014/  Still broken for me.

I think you should pay close attention to the pattern of indentation in `_config.yml`.  The way you've written the file seems to have a different pattern and number of spaces than the original did..  This indentation is very important.

Check the original again:
https://raw2.github.com/silshack/spring2014/gh-pages/_config.yml

And compare with yours:
https://raw2.github.com/zbay/spring2014/b1dc1c0d96c359548b36177ea02bc80bb4e8120d/_config.yml

It's important to verify that your site looks the way it should before sending a pull request.  We can talk more about this in class. For now, fix the indentation and verify it works and you should be good to go.

Nice Gravitar, by the way :)

zbay committed:
Attempted to fix my formatting problem.

Copied and pasted raw code for elliott and grant in an attempt to get my website operational. At this point, would I be better off editing the new _config.yml with everybody's information in it?

eah13 commented:
Nice!  I or grant will merge from the command line
